starcheck/calc_ccd_temps.py

```diff
@@ -195,10 +201,22 @@ def get_interval_temps(intervals, times, ccd_temp):
         stop_idx = 1 + np.searchsorted(times, interval['tstop'])
         start_idx = -1 + np.searchsorted(times, interval['tstart'])
         ok_temps = ccd_temp[start_idx:stop_idx]
+        itimes = times[start_idx:stop_idx]
         if len(ok_temps) > 0:
             obs['ccd_temp'] = np.max(ok_temps)
             obs['n100_warm_frac'] = dark_model.get_warm_fracs(
                 100, interval['tstart'], np.max(ok_temps))
+        if interval['obsid'] in obsreqs and len(ok_temps) > 0:
```

Comment:
Need `if obsreqs is None: obsreqs = {}` so this won't raise an exception for `obsreqs = None`.